### ISSUE TITLE
Consider separate packages for API and client

### ISSUE THREAD

**Issue by imalsogreg** [kobeyu/hasbitica#3]
Keeping the API and client under the same package makes it difficult to use the API in ghcjs applications. Which of course we may want to do :)

**Comment by tocklime** [comment]
Yes, I hadn't thought of that use. I'm currently using it to do some tasks in a cron job. I'll pull it apart soon, hopefully.
